packages/rn-tester-e2e/test/runners/buttonComponentScreenRunner.js

```diff
@@ -1,46 +1,86 @@
 import { defineFeature, loadFeature } from 'jest-cucumber';
-import { givenUserOnMainPage } from '../common_steps/givenUserOnMainPage.steps';
-import * as steps from '../steps/buttonComponentScreen.steps';
+import { userIsOnMainScreen, clickOkButton } from '../common_steps/common.steps';
+import componentsScreen from '../screenObjects/components.screen.js';
+import buttonComponentScreen from '../screenObjects/buttonComponent.screen.js';
 
-Object.entries(steps).forEach(([name, exported]) => window[name] = exported);
+// Object.entries(steps).forEach(([name, exported]) => window[name] = exported);
 const feature = loadFeature('test/features/buttonComponentScreen.feature');
 
+//Methods used more than once in the feature
+const buttonComponentShouldBeDisplayed = (then) => {
+    then('Verify that the Button component is displayed', async () => {
```

Comment:
Should we add "$" sign like on other methods?

Reply:
I would leave it as it is for now. If we will be checking another component, then we can refactor that method